server/views/pages/functionalSkills/macros/sortableTable.njk

```diff
@@ -0,0 +1,29 @@
+{#  -------------------------------------------------------------
+    Sortable table for the functional skills page
+    `config` argument is an object with the following properties:
+      * id - the id of the sortable table
+      * assessmentType - the name of the data array to loop through { englishSkills, mathsSkills, digitalSkills }
+    ------------------------------------------------------------- #}
+{% macro sortableTable(config) %}
+    <table class="govuk-table" data-module="moj-sortable-table" data-qa="{{ config.id }}-sortable-table">
+      <thead class="govuk-table__head">
+        <tr class="govuk-table__row">
+          <th scope="col" class="govuk-table__header">Name</th>
+          <th scope="col" class="govuk-table__header"{% if config.assessmentType.length > 1 %} aria-sort="none"{% endif %}>Location</th>
+          <th scope="col" class="govuk-table__header"{% if config.assessmentType.length > 1 %} aria-sort="desc"{% endif %}>Assessed on</th>
+          <th scope="col" class="govuk-table__header"{% if config.assessmentType.length > 1 %} aria-sort="none"{% endif %}>Level</th>
+        </tr>
+      </thead>
+
+      <tbody class="govuk-table__body">
+        {% for assesment in config.assessmentType %}
+        <tr class="govuk-table__row">
+          <td class="govuk-table__cell">{{ assesment.type | formatFunctionalSkillType }}</td>
+          <td class="govuk-table__cell">{{ assesment.prisonName }}</td>
```

Comment:
Do we handle the fallback for not finding a prison from the prison service in the controller code? Or should we handle it in the same way we do the Timeline?
This is from the timeline view, but you get the idea:
```
{{ event.prison.prisonName if event.prison.prisonName else event.prison.prisonId }}
```

Reply:
So if there isn't prison name in the `prisonService` it will fall back to the `prisonName` value on the skills assessment, but can add an extra fallback in the view to fallback to ID if there is neither of them!